### PR TITLE
Revert "Revert "New release - removed css-style-inject and fixed unit tests""

### PR DESCRIPTION
**Pull request by mario-subo**
Reverts sebgroup/react-components#757

### CODE REVIEW

File: lib/rollup.config.js [hunk 1]
```diff
@@ -6,6 +6,30 @@ import pkg from "./package.json";
 import commonPkg from "../package.json";
 
 const components = require("./src/index.json");
+
+function injectCSS(css, uniqueId, isPrepend) {
+    if (!css || !document) {
```
Comment (eweseong) [inline lib/rollup.config.js:11]: is the suggestion from Aaron no longer needed?

**Review by eweseong — APPROVED** [review]

File: lib/rollup.config.js (hunk 1 above)
Comment (mario-subo) [reply lib/rollup.config.js:11]: yeah it's still needed (and other stuff on develop) just making sure the history is back to what master used to be before I merge in the new changes haha

Comment (sjkayle) [inline lib/rollup.config.js:11]: Should use `typeof document === 'undefined'` as per [this PR](https://github.com/sebgroup/react-components/pull/756).

Comment (mario-subo) [reply lib/rollup.config.js:11]: https://github.com/sebgroup/react-components/pull/761#discussion_r943061367 :)

**Review by sjkayle — APPROVED** [review]

**Comment by mario-subo** [comment]
sorry for the confusion, this is a revert of a revert going directly to master :P

**Pull request merged**